safe/gui/tools/help/dock_help.py

```diff
@@ -91,20 +91,23 @@ def content():
         'There are three main areas to the dock panel:')))
     bullets = m.BulletedList()
     bullets.add(m.Text(
-        tr('the '),
-        m.ImportantText(tr(
-            'questions ')),
-        tr('area')))
+        # format 'the __questions__ area' for proper i18n
+        tr('the %s area') % (
+            m.ImportantText(tr(
+                'questions')).to_html(),
+        )))
```

Comment:
@ismailsunni can you review this?
I'm having trouble to translate it with previous format because the string is separated: 'the', 'questions', and 'area'.
With this, we can just translate 'the %s area' and reorder it in Bahasa Indonesia as : 'daerah %s'